CHANGELOG.md

```diff
@@ -9,10 +9,9 @@ and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## 5.1.4
```

Comment:
```suggestion
## [5.1.4]
```

We found out last time we released, that we need the square brackets here, even if it's no longer a link. (may as well change the one below as well)